src/sdl2_ui.cpp

```diff
@@ -268,6 +268,14 @@ bool Sdl2Ui::RefreshDisplayMode() {
 		SDL_GL_SetAttribute(SDL_GL_BLUE_SIZE, 5);
 		#endif
 
+		#if defined(__APPLE__) && defined(TARGET_OS_MAC)
```

Comment:
I'm not sure if we ever build for iOS, but I've done this instead of `defined(__APPLE__) && defined(__MACH__)` (which I found elsewhere in the project), because that will return true for both macOS and iOS. We want this behaviour _specifically_ on Macs. On iOS and other Apple platforms we definitely want to use the SDL default renderer instead.

Reply:
`TARGET_OS_MAC` can not be used for this purpose. It is defined as `0`.
http://sealiesoftware.com/blog/archive/2010/8/16/TargetConditionalsh.html

Reply:
`TARGET_OS_OSX` it is then

Reply:
Switched to `TARGET_OS_OSX` and removed the `defined` since it will be defined-but-zero on non-Mac Apple platforms